### ISSUE TITLE
Wait for the user's response

### ISSUE THREAD

**Issue by fedebisignani** [xmikos/hangupsbot#22]
Is there a way that within a command, the bot waits for an answer?

Example:

User command: get_forecast
Bot answer: Where?
User: "new york"

Everything within the same command

**Comment by wardellbagby** [comment]
No, not currently. It's an async loop, where every new message from a user is treated as the first message. There's no state or waiting logic currently in place.

This can be done, but it will be a fundamental change to how the bot handles messages.
